src/automaticassessmentconfiguration.cs

```diff
@@ -0,0 +1,81 @@
+//
+// AutomaticAssessmentConfiguration C# bindings
+//
+// Authors:
+//	Alex Soto <alexsoto@microsoft.com>
+//
+// Copyright (c) Microsoft Corporation.
+//
+
+using System;
+using Foundation;
+using ObjCRuntime;
+
+namespace AutomaticAssessmentConfiguration {
+
+	[ErrorDomain ("AEAssessmentErrorDomain")]
+	[Unavailable (PlatformName.MacCatalyst)]
+	[Mac (10,15,4), iOS (13,4)]
+	[Advice ("This API is not available when using UIKit on macOS.")]
+	[Native]
+	public enum AEAssessmentErrorCode : long {
+		Unknown = 1
+	}
+	
+	[Unavailable (PlatformName.MacCatalyst)]
+	[Mac (10,15,4), iOS (13,4)]
+	[Advice ("This API is not available when using UIKit on macOS.")]
+	[BaseType (typeof (NSObject))]
+	interface AEAssessmentConfiguration : NSCopying {
+
```

Comment:
hmm... does not look very helpful _yet_

Reply:
Agreed but nothing much we can do right now heh

Reply:
Did I _type_ that out loud ?
I was thinking of 🍎